pdns/dnsdistdist/docs/rules-actions.rst

```diff
@@ -1658,10 +1658,11 @@ The following actions exist.
   Before 1.7.0 this action was performed even when the query had been received over TCP, which required the use of :func:`TCPRule` to
   prevent the TC bit from being set over TCP transports.
 
-.. function:: TeeAction(remote[, addECS])
+.. function:: TeeAction(remote[, addECS[, local]])
 
```

Comment:
```suggestion
.. versionchanged:: 1.8.0
  Added the optional parameter ``local``.
```

Reply:
Commit in https://github.com/PowerDNS/pdns/pull/11889/commits/76fd1dd4505ff168c621d6543f27b9c54bb25222